src/css/App.css

```diff
@@ -708,7 +761,85 @@ a {
   left:0;
   bottom:0;
 }
-.read-more-button:hover:before,button:hover:after{
+/* .read-more-button:hover:before,button:hover:after{
   width:100%;
   transition:800ms ease all;
+} */
```

Comment:
@Etomovich, why kindly remove the commented out code.

Reply:
I will remove this as well

Reply:
@Etomovich Please implement this feedback